packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -451,6 +451,52 @@ void main() {
     expect(count, 0x010101);
   });
 
+  testWidgets('Nav bar static components respect MediaQueryData', (WidgetTester tester) async {
+    // This is a regression test for https://github.com/flutter/flutter/issues/174642
+    const double value = 10.0;
+
+    void expectCustomMediaQueryData(BuildContext context) {
+      expect(MediaQuery.platformBrightnessOf(context), Brightness.dark);
```

Comment:
This looks great, but is it ok if it doesn't check the `value` anymore?

Reply:
Yeah I think this is sufficient. Just a few is enough to verify our approach here, and thoroughly testing `copyWith` should be done in `MediaQueryData`'s tests.